managed/services/management/dbaas/kubernetes_server.go

```diff
@@ -424,6 +435,16 @@ func (k kubernetesServer) installDefaultOperators(operatorsToInstall map[string]
 			channel = "stable-v0"
 		}
 		operatorName := "victoriametrics-operator"
+		subs, err := kubeClient.GetSubscription(ctx, namespace, operatorName)
+
+		// If the VM operator already exists, we must delete it in order to reconfigure it.
+		// It migh be the case were the cluster is being re-registered and the server's IP has changed.
```

Comment:
Just out of curiosity but what about the option when we list existing agents running and if they are okay we do nothing? If we have less than 1 agent running then we can provision a new VM agent?